### ISSUE TITLE
Module does not compile on node v.12+

### ISSUE THREAD

**Issue by hillmanov** [nuodb/node-db-nuodb#27]
We really would like to update our application stack to the latest version on node, but this module is holding us back. Trying on anything greater than v0.10 results in the following output:

npm ERR! db-nuodb@2.0.2 install: `make build && make install`
npm ERR! Exit status 2
npm ERR!
npm ERR! Failed at the db-nuodb@2.0.2 install script 'make build && make install'.
npm ERR! This is most likely a problem with the db-nuodb package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     make build && make install
npm ERR! You can get their info via:
npm ERR!     npm owner ls db-nuodb
npm ERR! There is likely additional logging output above.

**Comment by joeycozza** [comment]
+1

**Comment by AchrafLansari** [comment]
+1

**Comment by PNE1** [comment]
+100000000000000000000000 we really need a driver for nodejs !

**Comment by joeycozza** [comment]
Yeah @PNE1 , we needed it so badly that we wrote the db querying portion in Python (there is a working Python module for Nuodb), and had our node app call that python script...  sure would have been nice to just use node!
